packages/EasyApiToken/composer.json

```diff
@@ -15,7 +15,9 @@
         "auth0/auth0-php": "^5.4",
         "psr/container": "^1.0",
         "laravel/lumen-framework": "^5.5",
-        "symfony/symfony": "^4.2"
```

Comment:
Not all projects are using symfony 5 yet, so instead we should set the version to `^4.2 || ^5.0`. And no need to require individual component it is only for dev.

Reply:
Must have been borked during merge. Fixed.